### PR TITLE
Bexelbie port fix

### PR DESCRIPTION
**Pull request by bexelbie**

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -12,7 +12,7 @@ Vagrant.configure(2) do |config|
 
   # Every Vagrant development environment requires a box. You can search for
   # boxes at https://atlas.hashicorp.com/search.
-  config.vm.box = "centos7-cdk-test"
+  config.vm.box = "centos/7"
```
Comment (navidshaikh) [inline Vagrantfile]: The box should be "atomicapp/dev"

**Comment by navidshaikh** [comment]
@bexelbie : The IP address returned after pulling this patch is correct (as verified using `ip a` inside the box), however using this IP address I can not ping to guest from host, and ping to `docker` daemon from Eclipse is also failing.

**Comment by bexelbie** [comment]
@navidshaikh I'll take a look. I only tested returns, not access.

**Comment by navidshaikh** [comment]
@bexelbie:  
I tried another scenario, added following line `Vagrantfile`

``` ruby
config.vm.network "private_network", type: "dhcp"
```

then

``` bash
vagrant reload
vagrant ssh

[vagrant@localhost ~]$ ip a
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN 
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
    inet6 ::1/128 scope host 
       valid_lft forever preferred_lft forever
2: eth0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc pfifo_fast state UP qlen 1000
    link/ether 52:54:00:cc:1a:1d brd ff:ff:ff:ff:ff:ff
    inet 10.0.2.15/24 brd 10.0.2.255 scope global dynamic eth0
       valid_lft 86084sec preferred_lft 86084sec
    inet6 fe80::5054:ff:fecc:1a1d/64 scope link 
       valid_lft forever preferred_lft forever
3: eth1: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc pfifo_fast state UP qlen 1000
    link/ether 08:00:27:92:a3:b2 brd ff:ff:ff:ff:ff:ff
    inet 172.28.128.3/24 brd 172.28.128.255 scope global dynamic eth1
       valid_lft 906sec preferred_lft 906sec
    inet6 fe80::a00:27ff:fe92:a3b2/64 scope link 
       valid_lft forever preferred_lft forever
4: docker0: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc noqueue state DOWN 
    link/ether 56:84:7a:fe:97:99 brd ff:ff:ff:ff:ff:ff
    inet 172.17.42.1/16 scope global docker0
       valid_lft forever preferred_lft forever
```

`docker` daemon is accessible (in Eclipse) by IP address `172.28.128.3` at port `2376` by `eth1` but not by IP address `10.0.2.15` by `eth0`. Also the box pingable by only `172.28.128.3`

**Comment by bexelbie** [comment]
This is what I am seeing too.  I am not sure where eth0 is coming from - I wonder if it is in the base box and needs to be removed.

**Comment by navidshaikh** [comment]
@bexelbie :  I think it is coming from [here](https://github.com/projectatomic/adb-atomic-developer-bundle/blob/master/build_tools/kickstarts/centos-7-kubernetes-vagrant.ks#L7),  the KS file.

@LalatenduMohanty  : any comments?

**Comment by LalatenduMohanty** [comment]
@navidshaikh I am confused . In the vagrantfile centos/7 is the guest box. Or it is adb?

**Comment by navidshaikh** [comment]
@LalatenduMohanty : It should be ADB, check my comment https://github.com/bexelbie/vagrant-adbinfo/pull/24/files#r44756569

**Comment by navidshaikh** [comment]
LGTM

**Pull request merged**